modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -20,7 +20,6 @@ You can install the AWS Load Balancer Operator by using the web console.
 
 . Navigate to *Operators* → *OperatorHub* in the {product-title} web console.
 . Select the *AWS Load Balancer Operator*. You can use the *Filter by keyword* text box or use the filter list to search for the AWS Load Balancer Operator from the list of Operators.
```

Comment:
We can/should add a step in here that indicates that you need to click the first install button on the pop up window to expand all the installation options and be able to complete the install. Like this:

```. Click "Install" to show all installation configuration options. ```

Reply:
Right, if you add a detail about clicking on `Install` button to expand the installation options, the "Select namespace" would start to make sense.